### PR TITLE
Fix Advanced option test, and responsiveness on withdraw and deposit page

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

- Fix testing of  AdvancedOption .
- mobile responsiveness on Widthdraw and Deposite page.

## Context.
closes https://github.com/saddle-finance/saddle-frontend/issues/826
## Screenshot

image

### CODE REVIEW

**Review by ug02fast — APPROVED** [review]
good fix sir

File: src/components/WithdrawPage.tsx [hunk 1]
```diff
@@ -85,99 +85,109 @@ const WithdrawPage = (props: Props): ReactElement => {
...
+                    </span>
+                  </div>
                 </div>
               </div>
             </div>
-          </div>
-          <AdvancedOptions />
-
-          <Stack direction="row" width="434px" pt={3} justifyContent="center">
+            <Box px={[3, 3, 0]} width="100%">
```
Comment (nkrulikovsky) [inline src/components/WithdrawPage.tsx:188]: This is good example of giving padding according to the break point in MUI.
For more detail, https://mui.com/system/basics/#responsive-values

Comment (ug02fast) [reply src/components/WithdrawPage.tsx:188]: Are breakpoints necessary to work on mobile or can pure Flexbox/Grid handle it?

Comment (nkrulikovsky) [reply src/components/WithdrawPage.tsx:188]: Pure Flexbox/Grid can handle it.
But for now the margin of the token Input is handled by scss, so now I gave padding according to the breakpoint as temporally.

**Review by hammeiam — APPROVED** [review]

**Pull request merged**